src/main/java/org/launchcode/techjobs/oo/Job.java

```diff
@@ -2,24 +2,132 @@
 
 import java.util.Objects;
 
-public class Job {
+//    private int id;
+//    private static int nextId = 1;
+//
+//    private String name;
+//    private Employer employer;
+//    private Location location;
+//    private PositionType positionType;
+//    private CoreCompetency coreCompetency;
+
+    // TODO: Add two constructors - one to initialize a unique ID and a second to initialize the
+    //  other five fields. The second constructor should also call the first in order to initialize
+    //  the 'id' field.
+
+    // TODO: Add custom equals and hashCode methods. Consider two Job objects "equal" when their id fields
+    //  match.
+
+
+    // TODO: Add getters for each field EXCEPT nextId. Add setters for each field EXCEPT nextID
+    //  and id.
+import java.util.Objects;
 
+public class Job {
+    //fields
     private int id;
     private static int nextId = 1;
+    private static final String DEFAULT_MESSAGE = "Data not available";
 
     private String name;
     private Employer employer;
     private Location location;
     private PositionType positionType;
     private CoreCompetency coreCompetency;
 
-    // TODO: Add two constructors - one to initialize a unique ID and a second to initialize the
-    //  other five fields. The second constructor should also call the first in order to initialize
-    //  the 'id' field.
+    //getters
+    public int getId() {
+        return id;
+    }
+    public String getName() {
+        return name;
+    }
+    public Employer getEmployer() {
+        return employer;
+    }
+    public Location getLocation() {
+        return location;
+    }
+    public PositionType getPositionType() {
+        return positionType;
+    }
+    public CoreCompetency getCoreCompetency() {
+        return coreCompetency;
+    }
 
-    // TODO: Add custom equals and hashCode methods. Consider two Job objects "equal" when their id fields
-    //  match.
+    //setters
+    public void setName(String name) {
+        this.name = name;
+    }
+    public void setEmployer(Employer employer) {
+        this.employer = employer;
+    }
+    public void setLocation(Location location) {
+        this.location = location;
+    }
+    public void setPositionType(PositionType positionType) {
+        this.positionType = positionType;
+    }
+    public void setCoreCompetency(CoreCompetency coreCompetency) {
+        this.coreCompetency = coreCompetency;
+    }
 
-    // TODO: Add getters for each field EXCEPT nextId. Add setters for each field EXCEPT nextID
-    //  and id.
-}
+    //constructors
```

Comment:
I would move your getters and setters here to place them underneath the associated comment.